docs/examples.rst

```diff
@@ -13,7 +13,7 @@ Now, to make basic measurement::
 
     import tcs34725
     from machine import I2C, Pin
-    i2c = I2C(Pin(5), Pin(4))
+    i2c = I2C(-1, Pin(5), Pin(4))
```

Comment:
Can you put that in a separate pull request, as it is not related to the feature you are adding?